integration/query_array_test.go

```diff
@@ -155,50 +130,8 @@ func TestQueryArrayDotNotation(t *testing.T) {
 		expectedIDs []any
 		err         *mongo.CommandError
 	}{
-		"PositionIndexGreaterThanArrayLength": {
-			filter:      bson.D{{"v.5", bson.D{{"$type", "double"}}}},
-			expectedIDs: []any{},
-		},
-		"PositionIndexAtTheEndOfArray": {
-			filter:      bson.D{{"v.1", bson.D{{"$type", "double"}}}},
-			expectedIDs: []any{"array-two"},
-		},
-
-		"PositionTypeNull": {
-			filter:      bson.D{{"v.0", bson.D{{"$type", "null"}}}},
```

Comment:
This one is removed but not added. I guess the case `DotNotationPositionTypeNull` should contain it.